examples/gno.land/r/demo/boards/post.gno

```diff
@@ -183,7 +183,12 @@ func (post *Post) RenderSummary() string {
 	return str
```

Comment:
Can we get .gno code to support syntax highlighting? By contacting github/microsoft?

Reply:
I already configured .gitattributes. Sometimes it works, not always.

the way to go is to wait for enough repos with .gno files, then open a PR against github/linguist so it becomes official globally. I regularly check for this.